src/findcdn/cdnengine/cdnengine.py

```diff
@@ -138,24 +126,27 @@ def grab_cdn(
                 for domain in newpot
             }
 
+            # Init completed ammount
+            completed: int = 0
             # Comb future objects for completed task pool.
             for future in concurrent.futures.as_completed(results):
                 try:
                     # Try and grab feature result to dequeue job
                     future.result(timeout=self.timeout)
-                except concurrent.futures.TimeoutError as e:
+                except concurrent.futures.TimeoutError as err:
                     # Tell us we dropped it. Should log this instead.
                     if self.interactive or self.verbose:
-                        print(f"Dropped due to: {e}")
+                        print(f"Dropped due to: {err}")
 
                 # Update status bar if allowed
                 if self.pbar:
-                    # We type ignore these as its "illegal" to access private attributes of an object
-                    pending = f"Pending: {executor._work_queue.qsize()} jobs"  # type: ignore
-                    threads = f"Threads: {len(executor._threads)}"  # type: ignore
+                    # We type ignore these as its "illegal" to access private attributes of an object but we need to (thanks python)
```

Comment:
The `type: ignore` jazz went away, but this comment is still here?

Reply:
Yeah I am not too sure why I left that. It is removed now in a11c66708586ca15d5b6ee8b7dfcd422cce77459 . Thank you for pointing that out!